LICENSE

```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2016 The Kubernetes Authors All Rights Reserved
+   Copyright [yyyy] [name of copyright owner]
```

Comment:
Shouldn't this be 
```
Copyright © 2021 SUSE  LLC. All Rights Reserved.
```
?

Reply:
No; this is an appendix section showing what your file header comments should look like.  This applies if we want to apply the license to specific files in the project.  See [upstream instructions](https://www.apache.org/licenses/LICENSE-2.0#apply).  The license itself doesn't care about the appendix.

Reply:
ah ok. I never read that part before.